src/bikeracks/helpers.py

```diff
@@ -100,6 +100,33 @@ def get_vote_status(database, rack_id, user_id):
     result = database.execute(query, (rack_id, user_id)).fetchone()
     return result
     
+def update_vote_count(database, rack_id, up_delta, down_delta):
+    query = """ UPDATE
+                    bikeracks
+                SET
+                    upvote_count=
+                    (
+                        SELECT
+                            MAX(upvote_count + ?, 0)
+                        FROM
+                            bikeracks
+                        WHERE
+                            rack_id=?
+                    ),
+                    downvote_count=
+                    (
+                        SELECT
+                            MAX(downvote_count + ?, 0)
+                        FROM
+                            bikeracks
+                        WHERE rack_id=?
+                    )
```

Comment:
I think these can be combined, possibly even without the extra `SELECT`. https://www.sqlite.org/lang_update.html

Does this work?
```sql
UPDATE bikeracks
SET (upvote_count, downvote_count) = (MAX(upvote_count + ?, 0), MAX(downvote_count + ?, 0))
WHERE rack_id = ?
```

Reply:
I tested it on the command line. Other stuff is broken in this branch so I couldn't test it that way (I just merged the PR that would take care of the problem and I didn't want to cause conflicts)

Reply:
I'm confused. What do you mean this branch is broken and you couldn't test it? You can't merge this if it's broken anyway, so what is the point making these changes?

Reply:
Merged the changes in master.